Standards/scs-XXXX-v1-volume-type-standard.md

```diff
@@ -0,0 +1,138 @@
+
+## Motivation
+
+As an SCS user, I want to be able to create volumes with certain common features, such as encryption or
+replication, and to do so in a standardized manner as well as programmatically.
+This standard outlines a way of formally advertising these common features for a volume type to
+non-privileged users, so that the most suitable volume type can be discovered and selected easily — both by
+the human user and by a program.
+
+## Design Considerations
+
+All Considerations can be looked up in detail in the [Decision Record for the Volume Type Standard.](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0111-v1-volume-type-decisions.md)
+
+### Systematic Description of Volume Types
+
+To test whether a deployment has volume types with certain aspects, the discoverability of the parameters in the volume type has to be given. As for the time this standard is created, there is no way for users to discover all aspects through OpenStack commands. Therefore the aspects, that are fulfilled within a volume type, should be stated in the beginning of the **description** of a volume type in the following manner:
+
+`[scs:aspect1, aspect2, ..., aspectN]...`
+
+There is no sorting of aspects required. Every aspect should only be mentioned to the maximal amount of one.
+
+### Standardized Aspects
+
+The following table shows, which aspects are considered in this standard. The last column shows how the description of the volume type has to be adjusted, if the aspect is fulfilled:
+
+| Aspect | Part of Standard | standardized description |
+| ---- | ---- | ------ |
+| Encryption | **Recommended** | **"[scs:encrypted]"** |
+| Replication | **Recommended** | **"[scs:replicated]"** |
+
+It is possible to use multiple of those aspects within one volume type. SCS will only ever look, if there is a volume type that has an aspect, but there don't have to be different volume types.
+Example: one volume type that uses LUKS-encryption with a ceph storage with inherent replication would fulfill all recommendations of this standard.
+
+## DEFAULT volume type
+
+There is always a default volume type defined in an OpenStack deployment. This volume type is created in the setup of cinder and will always be present in any OpenStack deployments under the name `__default__`. This standard does not have any requirements about this volume type at this moment, instead deployers are free to choose what fits best in their environment. Conversely, a cloud user can not expect any specific behavior or properties from volume types named `__default__`.
+
+The parameters of volume types described in this standard do not have to be applied to the chosen default volume type. And this standard will not make any assumptions about parameters being present in the default volume type.
+
+## REQUIRED volume types
+
+Currently, this standard will not require volume types with certain specification. This will change in the future.
```

Comment:
I don't see the point in this. We always say that today's recommendations are tomorrow's requirements, and therefore all tests that we build MUST emit a warning, instead of a mere information, when some recommendation is violated. So this is vapid and can be dropped.